src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -416,12 +416,13 @@ Domain<3> BinaryCompactObject::create_domain() const {
 
   Maps maps_first_outer_shell = sph_wedge_coordinate_maps<Frame::Inertial>(
       radius_enveloping_cube_, radius_enveloping_sphere_, 0.0, 1.0,
-      use_equiangular_map_, 0.0, true, 1.0, 2, {},
-      {domain::CoordinateMaps::Distribution::Linear}, ShellWedges::All);
+      use_equiangular_map_, 0.0, 1.0, 2, {},
```

Comment:
That's right, the polar axis for the bbh domain will be the x axis, but because that is not the change I'm making in this PR i'm leaving it unchanged for now